src/api/server.go

```diff
@@ -12,15 +12,9 @@ var (
 	// HTTP Connection timeouts for read/write
 	TIMEOUT = time.Second * 30
 
-	ServerData = serverData{
-		ApiUrl: "/api/v1/",
-	}
+	ROOT_URL = "/api/v1"
```

Comment:
I was just thinking about this. We probably should make this /api/v2 but I think this change is external to any of the PRs I opened today.